webservice/webservice.py

```diff
@@ -305,8 +306,11 @@ def get_manifest():
     """
     # Setup logging
     logger = logging.getLogger("dashboardService.webservice.get_manifest")
+    # Create query parameters.
     filters = request.args.get('filters', '{"file": {}}')
     logger.debug("Filters string is: {}".format(filters))
+    output_formats = request.args.get('output_formats', '')
```

Comment:
Did you see my comment about the name of the parameter in the [ticket](https://ucsc-cgl.atlassian.net/browse/AZUL-152?focusedCommentId=14263&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-14263)? I think you should just call this `format`.

It should not be plural, unless you process multiple formats on one call, which you do not.

Also, can you document at the top of method the query parameter and values.

Reply:
Agreed, I renamed it to `format`.